src/EventListener/AddLibrarySubscriber.php

```diff
@@ -23,7 +23,7 @@ public function add(RequestEvent $event): void
     {
         $request = $event->getRequest();
 
-        if (!$event->isMasterRequest() || !$this->isShopContext($request)) {
+        if (!$this->isMainRequest($event) || !$this->isShopContext($request)) {
```

Comment:
Doesn't work with SF4.4. You should revert these

Reply:
It does! You probably misread it, it's a BC layer (by using a private method).